elastic.go

```diff
@@ -190,9 +190,18 @@ func (es esClient) backupDocuments(sb *s3Backend, keyPath, indexGlob string) err
 			log.Fatalf("Could not open backup file for writing: %v", err)
 		}
 
-		key := getKey(keyPath)
+		log.Debug("Backup file ready for writing")
 
-		e, err := newEncryptor(key, wr)
+		privateKey, publicKeyList, err := getKeys(publicKeyPath)
+		if err != nil {
+			log.Error("Could not retrieve public key or generate private key")
+
+			return err
+		}
```

Comment:
```suggestion
		if err != nil {
			return err
		}
```
No point of logging an error here since `err` will be propagated to main and logged as `fatal`

Reply:
I think that it is good to have an identifier with the error in order to find easier were it fails but you are right on the other hand that it is good to avoid double errors, I have used the `return ftm.Errorf("whatever: %s", err)` option. I have fixed that in the entire file (even in cases that were not included in this PR)